src/components/Avatar/Avatar.tsx

```diff
@@ -30,9 +32,9 @@ export const Avatar: FunctionComponent<AvatarProps> = ({
 			})}
 		>
 			<AvatarIcon initials={initials} size={size} image={image} />
-			<div className="c-avatar-and-text__text">
-				{name && <p>{name}</p>}
-				{title && <p className="u-text-muted">{title}</p>}
+			<div>
+				{name && <p className={classnames({ dark })}>{name}</p>}
```

Comment:
Voor consistency in het project passen we dit best aan naar classNames ipv classnames voor die package name.

Reply:
ik denk dat we dit beter niet doen om duidelijk het verschil te houden tussen de package "classnames" en de property: "className"

Reply:
Voor mij is dit fine, maar ik denk dat we het overal anders gedaan hebben, dus misschien eens snel replace all'en naar degene die we willen gebruiken als dat niets breekt natuurlijk.

Reply:
Yes, ik gebruik ook al heel de tijd `classnames`

Reply:
new PR: https://github.com/viaacode/avo2-components/pull/178